WalletWasabi/Tor/TorSettings.cs

```diff
@@ -52,6 +52,7 @@ public TorSettings(string dataDir, string logFilePath)
 
 		/// <summary>Full path to executable file that is used to start Tor process.</summary>
 		public string TorPath { get; }
+
```

Comment:
Should there be blank lines between properties? I'm not sure sometimes there are sometimes there are not.

Reply:
I am not sure either, I think Codemaid added this blank line because we add a summary to the property.